models/contact.py

```diff
@@ -0,0 +1,119 @@
+import time
+
+from core.citext import CIText
+from core.db import db
+
+
+class Challenge(db.Model):
+    ## States
```

Comment:
Oh hm.. I guess I'm used to a manually ported version. I guess that's fine then. The values are a little weird, but w/e